clang/lib/CodeGen/CGExpr.cpp

```diff
@@ -2576,20 +2576,20 @@ LValue CodeGenFunction::EmitDeclRefLValue(const DeclRefExpr *E) {
     // Check for captured variables.
     if (E->refersToEnclosingVariableOrCapture()) {
 
-      // kitsune: if we are generating a kokkos-based lambda construct
```

Comment:
Looks like all the changes in this file are unrelated to reducer.

Reply:
Yeah, more changes coming related to this path -- I thought I stashed them for this PR but this  slipped through.  I'm tempted to leave as-is for now since it will just be part of the next PR anyways and this is minimal...